cc/transport/grpc_streaming_transport.h

```diff
@@ -17,17 +17,100 @@
 #ifndef CC_TRANSPORT_GRPC_STREAMING_TRANSPORT_H_
 #define CC_TRANSPORT_GRPC_STREAMING_TRANSPORT_H_
 
+#include <grpc/grpc.h>
+#include <grpcpp/channel.h>
+#include <grpcpp/client_context.h>
+#include <grpcpp/create_channel.h>
+#include <grpcpp/grpcpp.h>
+
+#include <memory>
+
 #include "absl/status/statusor.h"
 #include "absl/strings/string_view.h"
 #include "cc/transport/transport.h"
 #include "oak_remote_attestation/proto/v1/messages.pb.h"
+#include "oak_remote_attestation/proto/v1/service_streaming.grpc.pb.h"
+#include "oak_remote_attestation/proto/v1/service_streaming.pb.h"
 
 namespace oak::transport {
 
 class GrpcStreamingTransport : public TransportWrapper {
  public:
-  absl::StatusOr<::oak::session::v1::AttestationBundle> GetEvidence() override;
-  absl::StatusOr<std::string> Invoke(absl::string_view request_bytes) override;
+  explicit GrpcStreamingTransport(
+      std::unique_ptr<::grpc::ClientReaderWriter<::oak::session::v1::RequestWrapper,
+                                                 ::oak::session::v1::ResponseWrapper>>
+          channel_reader_writer)
+      : channel_reader_writer_(std::move(channel_reader_writer)) {}
```

Comment:
Why have the constructor accept a ClientReaderWriter instead of a stub?

Reply:
The original idea was to make `GrpcStreamingTransport` agnistic to the service definition. I.e. as long as it accepts `RequestWrapper` it would work.

But it looks like the instantiation of the template `ClientReaderWriter<RequestWrapper, ResponseWrapper>` is defined in the generated gRPC code `oak_remote_attestation/proto/v1/service_streaming.grpc.pb.h`